packages/cardano-services/test/Program/services/ogmios.test.ts

```diff
@@ -255,8 +255,8 @@ describe('Service dependency abstractions', () => {
           await serverClosePromise(ogmiosServer);
         });
 
-        it('ogmiosCardanoNode should not be a instance of Proxy ', () => {
-          expect(types.isProxy(ogmiosCardanoNode)).toEqual(false);
+        it('ogmiosCardanoNode should be a instance of Proxy ', () => {
+          expect(types.isProxy(ogmiosCardanoNode)).toEqual(true);
```

Comment:
Why `ogmiosCardanoNode` becomes an instance of proxy now since `ogmiosUrl` is used for the connection?

Reply:
Solved!